framework/tlvf/yaml/tlvf/WSC/eWscVendorExt.yaml

```diff
@@ -5,9 +5,11 @@ _namespace: WSC
 eWscVendorExtSubelementBssType:
   _type: enum
   BACKHAUL_STA: 0x80
+  FRONTHAUL_BACKHAUL_BSS: 0x60 # (FRONTHAUL_BSS | BACKHAUL_BSS)
```

Comment:
Same here, I think we should remove this commit completely. We don't need to define new values in bitmask enums.

Reply:
fixed